tests/lambda_endpoints/conftest.py

```diff
@@ -311,11 +311,22 @@ def _populate_file_catalog(session):
             ),
         ),
         AncillaryFiles(
-            file_path="/path/to/imap_swe_l1b-in-flight-cal_20231231_20240104_v002.cdf",
+            file_path="/path/to/imap_swe_l1b-in-flight-cal_20230102_v001.cdf",
             instrument="swe",
             descriptor="l1b-in-flight-cal",
-            start_date=datetime(2023, 12, 31),
-            end_date=datetime(2024, 1, 4),
+            start_date=datetime(2023, 1, 2),
+            version="v001",
+            extension="cdf",
+            ingestion_date=datetime.strptime(
+                "2024-01-25 23:35:26+00:00", "%Y-%m-%d %H:%M:%S%z"
+            ),
+        ),
+        AncillaryFiles(
+            file_path="/path/to/imap_swe_l1b-in-flight-cal_20240105_20250104_v002.cdf",
```

Comment:
Can you change to this? For a second, I thought you flipped the start and end date but realized later that you had different year.
```suggestion
            file_path="/path/to/imap_swe_l1b-in-flight-cal_20240104_20240106_v002.cdf",
```